tests/test_atoms.py

```diff
@@ -142,6 +142,7 @@ def test_list_quoted_str(a, quoting):
 
 
 @given(a=list_of_complex_numbers())
+@settings(deadline=400.0)
```

Comment:
When Hypothesis timeouts happen it usually means that there's something wrong with the tests themselves. With this project, I've seen it happen when the order of tokens in the grammar is "wrong" which results in longer parsing times. Does this happen locally too?

Reply:
Yes it also happens locally.